terraform/alerting/verification-server/alerts.tf

```diff
@@ -327,3 +327,44 @@ EOT
     google_logging_metric.stackdriver_export_error_count
   ]
 }
+
+# fast error budget burn alert
+resource "google_monitoring_alert_policy" "fast_burn" {
+  project      = var.monitoring-host-project
+  display_name = "Fast error budget burn"
+  combiner     = "OR"
+  enabled      = "true"
+  # create only if using GCLB, which means there's an SLO created
+  count = var.https-forwarding-rule == "" ? 0 : 1
```

Comment:
I think there's a better variable (it might be a local) to key off of.

Reply:
There was a `locals.enable_lb` but that was removed a while back. We need to have `var.https-forwarding-rule` anyway because we need it in the backend latency alert.

Reply:
Per @yegle : There was a locals.enable_lb but that was removed a while back. We need to have var.https-forwarding-rule anyway because we need it in the backend latency alert.